cdap-app-fabric/src/main/java/co/cask/cdap/metadata/profile/ProfileMetadataMessageProcessor.java

```diff
@@ -48,18 +47,14 @@
 import co.cask.cdap.proto.id.ProfileId;
 import co.cask.cdap.proto.id.ProgramId;
 import co.cask.cdap.proto.id.ScheduleId;
-import co.cask.cdap.proto.id.WorkflowId;
 import co.cask.cdap.store.NamespaceMDS;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
Don't use `*` in import. I think our checkstyle will fail as well.

Reply:
Yep sorry - still trying to get the correct settings in my IDE